dhis-2/dhis-api/src/main/java/org/hisp/dhis/category/CategoryService.java

```diff
@@ -509,6 +526,9 @@ CategoryOptionCombo getCategoryOptionCombo(
 
   List<CategoryOptionGroup> getCategoryOptionGroups(CategoryOptionGroupSet groupSet);
 
+  List<CategoryOptionGroup> getCategoryOptionGroupByCategoryOption(
```

Comment:
I think at least for services we should move to IDs as arguments. I know we have plenty of code that uses object parameters but we have to start somewhere.